pod/perlhack.pod

```diff
@@ -763,18 +763,22 @@ tested.  Tests in F<t/opbasic>, for instance, have been placed there
 rather than in F<t/op> because they test functionality which
 F<t/test.pl> presumes has already been demonstrated to work.
 
-=item * F<t/cmd>, F<t/run>, F<t/io> and F<t/op>
+=item * All other subdirectories of F<t/>
 
 Now that basic require() and subroutines are tested, you can use the
 F<t/test.pl> library.
 
-You can also use certain libraries like Config conditionally, but be
+You can also use certain libraries like F<Config> conditionally, but be
 sure to skip the test gracefully if it's not there.
```

Comment:
`Config` is a module with documentation, this could be a L<> instead.  Other than that is looks good to me.